src/vuex-jsonapi.js

```diff
@@ -23,23 +23,25 @@ const handleError = commit => errorResponse => {
   throw errorResponse;
 };
 
+const initialState = () => ({
+  records: [],
+  related: [],
+  filtered: [],
+  page: [],
+  error: null,
+  status: STATUS_INITIAL,
+  links: {},
+  lastCreated: null,
+  lastMeta: null,
+});
+
 const resourceModule = ({ name: resourceName, httpClient }) => {
   const client = new ResourceClient({ name: resourceName, httpClient });
 
   return {
     namespaced: true,
 
-    state: {
-      records: [],
-      related: [],
-      filtered: [],
-      page: [],
-      error: null,
-      status: STATUS_INITIAL,
-      links: {},
-      lastCreated: null,
-      lastMeta: null,
-    },
+    state: initialState,
```

Comment:
I think this line is assigning the `initialState` function to `state`, rather than running the function to get the initial state data (as you do in `RESET_STATE` on line 111). It might actually still work if Vuex is okay with just assigning state values to a function! But not what intended =]

Reply:
Yes, that is correct. I will push a fix for it